python/modem-firmware-1.3+/device_credentials_installer.py

```diff
@@ -551,12 +559,24 @@ def main():
 
     # initialize colorama
     if is_windows:
-        init(convert = not plain)
+        init_colorama()
 
     if verbose:
         print(local_style('OS detect: Linux={}, MacOS={}, Windows={}'.
                           format(is_linux, is_macos, is_windows)))
 
+    if args.cmd_type == CMD_TYPE_TLS_SHELL and not args.local_cert:
+        print(error_style(f"cmd_type '{CMD_TYPE_TLS_SHELL}' requires --local_cert"))
```

Comment:
Yes! Once that is added to the TLS credentials shell, then this can be updated here

I could insert the word "currently" if you think we should allude to that

Reply:
Sure, or add a comment to the code to that effect.